addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/ec2-linux-instance.cfn.yml

```diff
@@ -269,6 +269,8 @@ Resources:
           # Signal result to CloudFormation
           /opt/aws/bin/cfn-signal -e $? --stack "${AWS::StackName}" --resource "EC2Instance" --region "${AWS::Region}"
 
+          sudo su - ec2-user
```

Comment:
Why need this command?

Reply:
Currently Service Workbench implementation, mount s3 data script will be executed when ec2-user login. If we use ssm-user to login ec2 linux instance, data will not be mounted until change to ec2-user via `sudo su - ec2-user` command. So I added this command to make data be mounted during the Workspace is created.